tsconfig.stories.json

```diff
@@ -13,6 +13,7 @@
     "strictNullChecks": false,
     "noUnusedLocals": false,
     "types": [],
-    "incremental": false
+    "incremental": false,
+    "skipLibCheck": true
```

Comment:
This fixes the CI failures, is it ok to add?

Reply:
This seems to be a conflict with the formik 2.x and the version of @types/react we are using, I tried moving to @types/react 16.9.12 but that introduced a bunch of conflicts with react-testing library, updating that caused issues with enzyme. So wondering if we should hold off for now and maybe revisit this later, perhaps after enzyme gets removed.

Reply:
Ugh. There is still a few tests written in Enzyme, `Menu` being the most complicated. The other files have only a few tests each. If we add this, I'd like an issue to get rid of it again.